src/user/configure-metadata.md

```diff
@@ -2774,7 +2774,9 @@ Table: Validation Rule functions
 | log | (expression [, base ]) | Returns the natural logarithm (base e) of the numeric expression. If an integer is given as a second argument, returns the logarithm using that base. |
 | log10 | (expression) | Returns the common logarithm (base 10) of the numeric expression. |
 | orgUnit.ancestor | (orgUnitUid [, orgUnitUid ...]) | Returns true if the organisation unit is a descendant of any of the (1 or more) organisation units, otherwise false. |
+| orgUnit.dataSet | (dataSetUid [, dataSetUid ...]) | Returns true if the organisation unit is assgined to of any of the (1 or more) data sets, otherwise false. |
```

Comment:
Just small typos:
`assgined`
`to of any of`

The same applies to the other sentences below.